code/modules/clothing/head/helmet.dm

```diff
@@ -102,7 +102,7 @@
 	desc = "An extremely robust, space-worthy helmet in a nefarious red and black stripe pattern."
 	icon_state = "swatsyndie"
 	item_state = "swatsyndie"
-	armor = list(melee = 40, bullet = 30, laser = 30,energy = 30, bomb = 50, bio = 90, rad = 20)
+	armor = list(melee = 55, bullet = 40, laser = 35,energy = 40, bomb = 55, bio = 90, rad = 20)

```

Comment:
we talked about this, steelpoints buff was retarded, 
If I make a PR to lower that will you megre it?